ui/js/models/perf/alerts.js

```diff
@@ -25,16 +25,16 @@ treeherder.factory('PhAlerts', [
 
             // for talos only, automatically add related branches
             if (performanceFrameworkId === 1) {
-                _.forEach(thPerformanceBranches, function(performanceBranch) {
-                    if (performanceBranch !== alertRepository) {
-                        url += "&series=[" + [performanceBranch, signature, 0] + "]";
-                    }
-                });
                 if (alertRepository === "mozilla-beta") {
-                    url += "&series=[" + ["mozilla-aurora", signature, 0] + "]";
+                    url += "&series=[" + ["mozilla-inbound", signature, 0] + "]";
+                } else {
+                    _.forEach(thPerformanceBranches, function(performanceBranch) {
+                        if (performanceBranch !== alertRepository) {
+                            url += "&series=[" + [performanceBranch, signature, 0] + "]";
+                        }
+                    });
```

Comment:
for downstream, it is a big mess looking at all the data points of inbound/autoland since the time range is almost always larger- we should be able to see the alert as either the root or the downstream on inbound only.